cmd/queue/main.go

```diff
@@ -322,8 +333,51 @@ func main() {
 		logger.Fatalw("Failed to parse localhost URL", zap.Error(err))
 	}
 
+	queueProxyL3 := fmt.Sprintf("%s:%d", env.ServingPod, networking.ServiceHTTPPort)
+	zipkinEndpoint, err := zipkin.NewEndpoint(env.ServingPod, queueProxyL3)
+
+	if err != nil {
+		logger.Error("Unable to create tracing endpoint")
```

Comment:
```suggestion
		logger.Fatalw("Unable to create tracing endpoint", zap.Error(err))
```

And drop the return below, Fatal will exit anyway.